src/general/ModLoader.cs

```diff
@@ -0,0 +1,320 @@
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        confirmationPopup.PopupCentered();
+    }
+
+    private void ResetGame()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (!File.Exists(Directory.GetCurrentDirectory() + "/Thrive.pck"))
+        {
+            GD.Print("Fail to find Thrive");
+            return;
+        }
+
+        if (ProjectSettings.LoadResourcePack(Directory.GetCurrentDirectory() + "/Thrive.pck", true))
+        {
+            GD.Print("Reset successful");
+        }
+        else
+        {
+            GD.Print("Reset failed");
+        }
+
+        SceneManager.Instance.ReturnToMenu();
+    }
+
+    private void OnMoveUpPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (loadedItemList.GetSelectedItems().Length > 0)
+        {
+            var selectedItemIndex = loadedItemList.GetSelectedItems()[0];
+            if (selectedItemIndex == 0)
+            {
+                return;
+            }
+
+            loadedItemList.MoveItem(selectedItemIndex, selectedItemIndex - 1);
+            selectedItemIndex = loadedItemList.GetSelectedItems()[0];
+            MoveItem(loadedModList, selectedItemIndex + 1, selectedItemIndex);
+        }
+        else if (unloadedItemList.GetSelectedItems().Length > 0)
+        {
+            var selectedItemIndex = unloadedItemList.GetSelectedItems()[0];
+
+            if (unloadedItemList.GetSelectedItems()[0] == 0)
+            {
+                return;
+            }
+
+            unloadedItemList.MoveItem(selectedItemIndex, selectedItemIndex - 1);
+            selectedItemIndex = unloadedItemList.GetSelectedItems()[0];
+            MoveItem(modList, selectedItemIndex + 1, selectedItemIndex);
+        }
+    }
+
+    private void OnMoveDownPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        if (loadedItemList.GetSelectedItems().Length > 0)
+        {
+            var selectedItemIndex = loadedItemList.GetSelectedItems()[0];
+            if (selectedItemIndex == loadedItemList.GetItemCount() - 1)
+            {
+                return;
+            }
+
+            loadedItemList.MoveItem(selectedItemIndex, selectedItemIndex + 1);
+            selectedItemIndex = loadedItemList.GetSelectedItems()[0];
+            MoveItem(loadedModList, selectedItemIndex, selectedItemIndex - 1);
+        }
+        else if (unloadedItemList.GetSelectedItems().Length > 0)
+        {
+            var selectedItemIndex = unloadedItemList.GetSelectedItems()[0];
+            if (selectedItemIndex == unloadedItemList.GetItemCount() - 1)
+            {
+                return;
+            }
+
+            unloadedItemList.MoveItem(selectedItemIndex, selectedItemIndex + 1);
+            selectedItemIndex = unloadedItemList.GetSelectedItems()[0];
+            MoveItem(modList, selectedItemIndex, selectedItemIndex - 1);
+        }
+    }
+
+    private void MoveItem(List<ModInfo> list, int oldIndex, int newIndex)
```

Comment:
Is there no inbuilt method for this?
What about using a LINQ sort on the list to get it to the wanted order?